### DISCUSSION TITLE
Python project-config deprecated?

### DISCUSSION THREAD

**Discussion by ZeevR** [vercel/community#1289] [Help] [answered]
### Summary

It states here: https://vercel.com/docs/project-configuration#legacy/routes that the config used in vercel.json in all of the examples is deprecated in favor of the functions syntax.
Is it true for python as well? Is there an example? 

BTW
Is there a complete guide detailing best practices and a good dev workflow when working with vercel and python?

### Example

https://github.com/vercel/examples/blob/main/python/flask/vercel.json

**Comment by amyegan** [comment, marked as answer]
There should be no problem with using `routes` and `builds` in your Python app for now. But you could upgrade to `rewrites` if you want to use the most current configuration option. I have an example using the existing Flask example with the `vercel.json` modified to use `rewrites` (https://github.com/amyegan/flask-example/blob/247f323909a1089c56d4cc87d6d098afbe687a2b/vercel.json) which you can see deployed at https://flask-example-iota.vercel.app/

**Comment by ZeevR** [comment]
Amazing! 🙏
